Modules/Bridge/VtkGlue/include/vtkCaptureScreen.h

```diff
@@ -51,11 +51,10 @@ class vtkCaptureScreen
     Capture(m_Renderer, iFileName);
   }
 
-private:
-  vtkCaptureScreen(const vtkCaptureScreen &) = delete;
   void
   operator=(const vtkCaptureScreen &) = delete;
 
+private:
```

Comment:
moving `private:` label here makes `operator=(const vtkCaptureScreen &)` public.

Reply:
YES!  deleted functions are supposed to be public!

@mseng10 In cases where both copy constructor and opereator= are deleted, the DISSALLOW MACRO should be used.